unit-tests/algo/d2rgb/compare-to-bin-file.h

```diff
@@ -413,7 +414,26 @@ bool compare_to_bin_file(
 {
     TRACE("Comparing " << filename << " ...");
     bool ok = true;
-    auto obj_matlab = read_from< D >(bin_dir(scene_dir) + filename);
+    auto obj_matlab = read_from< D >( join( bin_dir( scene_dir ), filename ) );
 
     return compare_t(obj_matlab, obj_cpp);
+}
+
+bool read_thermal_data( std::string dir,
+                              double hum_temp,
+                              const std::pair< double, double > & in_fx_fy,
```

Comment:
in_fx_fy isn't used

Reply:
not after I removed the - correct_thermal_scale